.travis.yml

```diff
@@ -24,5 +29,5 @@ deploy:
   file_glob: true
   on:
     tags: true
-    repo: appsody/stacks
-    branch: master
\ No newline at end of file
+    repo: kabanero-io/collections
+    #    branch: master
```

Comment:
Could we just remove this line rather than comment it out? Alternatively add a few words around when it should be restored if required for any reason.

Reply:
Will fix up later as I have more changes to go into this file soon.